Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1950,15 +2145,22 @@ def main():
         'exabeam-get-notable-assets': get_notable_assets,
         'exabeam-get-notable-sequence-details': get_notable_sequence_details,
         'exabeam-get-notable-session-details': get_notable_session_details,
-        'exabeam-get-sequence-eventtypes': get_notable_sequence_event_types
+        'exabeam-get-sequence-eventtypes': get_notable_sequence_event_types,
+        'exabeam-list-incident': list_incidents,
     }
 
     try:
         client = Client(base_url.rstrip('/'), verify=verify_certificate, username=username,
                         password=password, proxy=proxy, headers=headers)
         command = demisto.command()
         LOG(f'Command being called is {command}.')
-        if command in commands:
+        if command == 'fetch-incidents':
+            incidents, next_run = fetch_incidents(client, demisto.params())
+            demisto.setLastRun(next_run)
+            demisto.incidents(incidents)
+        elif command == 'test-module':
+            return_outputs(*commands[command](client, demisto.args(), demisto.params()))  # type: ignore
```

Comment:
why the ignore? If absolutely necessary, specify what to ignore. Otherwise, avoid it.